src/admin-views/zapier/dashboard/endpoints/endpoint.php

```diff
@@ -19,80 +19,101 @@
-			] );
-		}
+			]
+		);
+	}
 	?>
 	<?php
-		$queue_classes = [ 'tec-automator-grid-item', 'tec-settings-connection-endpoint-dashboard-details__queue-wrap' ];
-		$queue_label   = _x( 'Queue', 'Label for the Zapier Endpoint Dashboards endpoint queue.', 'event-automator' );
-		if ( $endpoint['type'] === 'queue' && $endpoint['enabled'] ) {
-			$this->template( 'zapier/api/components/read-only', [
+	$queue_classes = [ 'tec-automator-grid-item', 'tec-settings-connection-endpoint-dashboard-details__queue-wrap' ];
+	$queue_label   = _x( 'Queue', 'Label for the Zapier Endpoint Dashboards endpoint queue.', 'event-automator' );
+	if ( $endpoint['type'] === 'queue' && $endpoint['enabled'] ) {
+		$this->template(
+			'zapier/api/components/read-only',
+			[
 				'classes_wrap'  => $queue_classes,
 				'label'         => $queue_label,
 				'screen_reader' => _x( 'The Queue for the Zapier endpoint.', 'The screen reader text of the label for the Zapier Endpoint Dashboard endpoint queue.', 'event-automator' ),
-				'id'            => "tec_automator_zapier_endpoint_queue_" . $endpoint['id'],
+				'id'            => 'tec_automator_zapier_endpoint_queue_' . $endpoint['id'],
 				'name'          => "tec_automator_zapier[]['endpoint_queue']",
 				'value'         => $endpoint['count'],
-			] );
-		} else {
-			$this->template( 'zapier/dashboard/components/disabled', [
+			]
+		);
+	} else {
+		$this->template(
+			'zapier/dashboard/components/disabled',
+			[
 				'classes_wrap'  => $queue_classes,
 				'label'         => $queue_label,
 				'screen_reader' => _x( 'The Queue is disabled for this Zapier endpoint.', 'The screen reader text of the label for the Zapier Endpoint Dashboard endpoint queue when disabled.', 'event-automator' ),
-			] );
-		}
+			]
+		);
+	}
 	?>
 	<div class="tec-automator-grid-item tec-settings-connection-endpoint-dashboard-details__actions-wrap tec-common-zapier-details__actions-wrap">
 		<?php
-			$this->template( 'zapier/dashboard/components/clear-button', [
-				'endpoint' => $endpoint,
-				'manager'  => $manager,
-				'url'      => $url,
-			] );
-			$this->template( 'zapier/dashboard/components/status-button', [
-				'endpoint' => $endpoint,
-				'manager'  => $manager,
-				'url'      => $url,
-			] );
-
-			if ( $endpoint['missing_dependency'] ) {
-				$this->template( 'zapier/dashboard/components/missing-dependency', [
+		if ( $endpoint['missing_dependency'] ) {
+			$this->template(
+				'zapier/dashboard/components/missing-dependency',
+				[
 					'endpoint' => $endpoint,
 					'manager'  => $manager,
 					'url'      => $url,
-				] );
-			}
+				]
+			);
+		} else {
+			$this->template(
+				'zapier/dashboard/components/clear-button',
+				[
+					'endpoint' => $endpoint,
+					'manager'  => $manager,
+					'url'      => $url,
+				]
+			);
+			$this->template(
+				'zapier/dashboard/components/status-button',
+				[
+					'endpoint' => $endpoint,
+					'manager'  => $manager,
+					'url'      => $url,
+				]
+			);
+		}
```

Comment:
Same comment here from above about putting the template args into a variable.

Reply:
@codingmusician thanks. To give you some context, this is [a file from EVA](https://github.com/the-events-calendar/event-automator/blob/main/src/admin-views/dashboard/endpoints/endpoint.php) that was merged in Common recently. I did not make these changes; all these additions you see are because of the PHPCS formating to satisfy the @tec-bot review. 

I had [different commits](https://github.com/the-events-calendar/tribe-common/pull/2101/commits) for my changes and for the code formatting to help the reviewers.

The actual change in this file was [at line 97 of the original](https://github.com/the-events-calendar/event-automator/blob/main/src/admin-views/dashboard/endpoints/endpoint.php#L97C4-L97C44), where it had a 
`if ( $endpoint['missing_dependency'] ) {`

I moved that higher to also exclude the action buttons from showing. Everything else is from the phpcbf.